Eliminating some instances of 'should', 'must' (#30)
AMWA-TV/nmos-authorization-implementation-guide: docs/4.3. Controller to Authorization Server Interactions.md

```

## Refreshing Bearer Tokens
The Bearer Token has a limited life specified by the value of the `expires_in` parameter in seconds. The Controller should refresh the token before [it exceeds its half life](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime) (for example if the token lifetime is 30 seconds, then it should be refreshed at least 15 seconds before token expiry).

> Because refresh tokens have the potential for a long lifetime, developers should ensure that they are stored securely to keep them from being leaked.

Reviewer: Perhaps:
>Because refresh tokens have the potential for a long lifetime, it is important that they are stored securely to keep them from being leaked.

to avoid the use of 'should'?

Reviewer 2: ```suggestion
Because refresh tokens can have a long lifetime, it is important that they are stored securely to prevent them from being leaked.
```
